implementations/rust/ockam/ockam_command/src/vault/delete.rs

```diff
@@ -18,11 +20,14 @@ after_long_help = docs::after_help(AFTER_LONG_HELP)
 )]
 pub struct DeleteCommand {
     /// Name of the vault
-    pub name: String,
+    pub name: Option<String>,
 
     /// Confirm the deletion without prompting
     #[arg(display_order = 901, long, short)]
     yes: bool,
+
+    #[arg(long, short, group = "vaults")]
```

Comment:
Is the `groups` attribute needed here?

Reply:
I've putted in all the PRs this because I was thinking that after the refactoring was needed for other features, basically copied from the node implementation.